static/css/restyle/restyle.less

```diff
@@ -1970,23 +1970,19 @@ body.editor-tools {
   }
 
   div.section {
-    ul.tabnav li.selected a {
-      padding-bottom: 5px;
-      background-color: #ffffff;
-      border-color: #ebebeb;
-      color: @default-font-color;
-    }
-
-    ul.tabnav li a {
-      background-color: #ecf5fe;
-      border-top-left-radius: 0;
-      border-top-right-radius: 0;
-      color: @link-on-white-bg;
-    }
-
-    a:hover {
```

Comment:
Is removing this hover effect intended? (I'm guessing yes, as a hover that changes `padding` is weird, but just confirming)